ADVANCED-USAGE-README.md

```diff
@@ -0,0 +1,62 @@
+# PatternFly React
+This README covers advanced usage topics for PatternFly React users.  
+
+**Community:** [PatternFly website](https://www.patternfly.org) | [Slack](https://slack.patternfly.org) | [Medium](https://medium.com/patternfly) | [Mailing list](https://www.redhat.com/mailman/listinfo/patternfly)
```

Comment:
Do we need to include the community part again when it's already in the root README?

Reply:
Yes  on the community part - these readme's are visible from the npm package page (not just in github) and quick links to important details from that location are critical.

Reply:
I think you are right on this one - removing it...